### PR TITLE
🌐 Add Korean translation for `docs/ko/docs/benchmarks.md`

### PR DESCRIPTION
**Pull request by Limsunoh**
~~Started translation of `docs/ko/docs/benchmarks.md`~~
Add Korean translation for `docs/ko/docs/benchmarks.md`

### CODE REVIEW

File: docs/ko/docs/benchmarks.md [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+
+계층 구조는 다음과 같습니다:
+
+* **Uvicorn**: ASGI 서버
+    * **Starlette**: (Uvicorn 사용) 웹 마이크로 프레임워크
+        * **FastAPI**: (Starlette 사용) API 구축을 위한 데이터 검증 등 여러 추가 기능이 포함된 API 마이크로 프레임워크
+
+* **Uvicorn**:
+    * 서버 자체 외에는 많은 추가 코드가 없기 때문에 최고의 성능을 발휘합니다.
+    * 직접 Uvicorn으로 애플리케이션을 작성하지는 않을 것입니다. 그렇게 한다면 Starlette(또는 **FastAPI**)가 제공하는 모든 코드를 포함해야 하기 때문입니다. 그리고 그렇게 만든다면 최종 애플리케이션은 프레임워크를 사용하는 것과 동일한 오버헤드를 가지며 앱 코드와 버그가 최소화됩니다.
```
Comment (JaeHyuckSa) [inline docs/ko/docs/benchmarks.md:23]:
```suggestion
    * 직접 Uvicorn으로 애플리케이션을 작성하지는 않을 것입니다. 즉, 사용자의 코드에는 적어도 Starlette(또는 **FastAPI**)에서 제공하는 모든 코드가 포함되어야 합니다. 그렇게 하면 최종 응용 프로그램은 프레임워크를 사용하고 앱 코드와 버그를 최소화하는 것과 동일한 오버헤드를 갖게 됩니다.
```

File: docs/ko/docs/benchmarks.md [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+    * Uvicorn을 비교할 때는 Daphne, Hypercorn, uWSGI 등의 애플리케이션 서버와 비교하세요.
+* **Starlette**:
+    * Uvicorn 다음으로 좋은 성능을 발휘합니다. 사실 Starlette는 Uvicorn을 사용하여 실행됩니다. 따라서 더 많은 코드를 실행해야 하기 때문에 Uvicorn보다 "느려질" 수밖에 없습니다.
+    * 하지만 경로 기반 라우팅 등 간단한 웹 애플리케이션을 구축할 수 있는 도구를 제공합니다.
+    * Starlette를 비교할 때는 Sanic, Flask, Django 등의 웹 프레임워크(또는 마이크로 프레임워크)와 비교하세요.
+* **FastAPI**:
+    * Starlette가 Uvicorn을 사용하고 Uvicorn보다 빨라질 수 없는 것과 마찬가지로, **FastAPI**는 Starlette를 사용하므로 더 빠를 수 없습니다.
+    * FastAPI는 Starlette 위에 더 많은 기능을 제공합니다. API를 구축할 때 거의 항상 필요한 데이터 검증 및 직렬화와 같은 기능이 포함되어 있습니다. 그리고 이를 사용하면 자동 문서화 기능도 제공됩니다(자동 문서화는 애플리케이션 실행 시 오버헤드를 추가하지 않고 시작 시 생성됩니다).
```
Comment (JaeHyuckSa) [inline docs/ko/docs/benchmarks.md:31]:
```suggestion
    * FastAPI는 Starlette에 추가적으로 더 많은 기능을 제공합니다. API를 구축할 때 거의 항상 필요한 데이터 검증 및 직렬화와 같은 기능이 포함되어 있습니다. 그리고 이를 사용하면 자동 문서화 기능도 제공됩니다(자동 문서화는 애플리케이션 실행 시 오버헤드를 추가하지 않고 시작 시 생성됩니다).
```

**Review by JaeHyuckSa — COMMENTED** [review]
Hello! @Limsunoh, thank you for your review. 👍 There are a few revisions needed.

File: docs/ko/docs/benchmarks.md [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+계층 구조는 다음과 같습니다:
+
+* **Uvicorn**: ASGI 서버
+    * **Starlette**: (Uvicorn 사용) 웹 마이크로 프레임워크
+        * **FastAPI**: (Starlette 사용) API 구축을 위한 데이터 검증 등 여러 추가 기능이 포함된 API 마이크로 프레임워크
+
+* **Uvicorn**:
+    * 서버 자체 외에는 많은 추가 코드가 없기 때문에 최고의 성능을 발휘합니다.
+    * 직접 Uvicorn으로 애플리케이션을 작성하지는 않을 것입니다. 그렇게 한다면 Starlette(또는 **FastAPI**)가 제공하는 모든 코드를 포함해야 하기 때문입니다. 그리고 그렇게 만든다면 최종 애플리케이션은 프레임워크를 사용하는 것과 동일한 오버헤드를 가지며 앱 코드와 버그가 최소화됩니다.
+    * Uvicorn을 비교할 때는 Daphne, Hypercorn, uWSGI 등의 애플리케이션 서버와 비교하세요.
```
Comment (JaeHyuckSa) [inline docs/ko/docs/benchmarks.md:24]:
```suggestion
    * Uvicorn을 비교할 때는 Daphne, Hypercorn, uWSGI 등의 응용 프로그램 서버와 비교하세요.
```

File: docs/ko/docs/benchmarks.md [hunk 4]
```diff
@@ -0,0 +1,34 @@
...
+* **Starlette**:
+    * Uvicorn 다음으로 좋은 성능을 발휘합니다. 사실 Starlette는 Uvicorn을 사용하여 실행됩니다. 따라서 더 많은 코드를 실행해야 하기 때문에 Uvicorn보다 "느려질" 수밖에 없습니다.
+    * 하지만 경로 기반 라우팅 등 간단한 웹 애플리케이션을 구축할 수 있는 도구를 제공합니다.
```
Comment (JaeHyuckSa) [inline docs/ko/docs/benchmarks.md:27]:
```suggestion
    * 하지만 경로 기반 라우팅 등 간단한 웹 응용 프로그램을 구축할 수 있는 도구를 제공합니다.
```

File: docs/ko/docs/benchmarks.md (hunk 1 above)
Comment (JaeHyuckSa) [inline docs/ko/docs/benchmarks.md:23]:
```suggestion
    * 직접 Uvicorn으로 응용 프로그램을 작성하지는 않을 것입니다. 즉, 사용자의 코드에는 적어도 Starlette(또는 **FastAPI**)에서 제공하는 모든 코드가 포함되어야 합니다. 그렇게 하면 최종 응용 프로그램은 프레임워크를 사용하고 앱 코드와 버그를 최소화하는 것과 동일한 오버헤드를 갖게 됩니다.
```

File: docs/ko/docs/benchmarks.md (hunk 3 above)
Comment (JaeHyuckSa) [inline docs/ko/docs/benchmarks.md:24]:
```suggestion
    * Uvicorn을 비교할 때는 Daphne, Hypercorn, uWSGI 등의 응용 프로그램 서버와 비교하세요.
```

File: docs/ko/docs/benchmarks.md (hunk 4 above)
Comment (JaeHyuckSa) [inline docs/ko/docs/benchmarks.md:27]:
```suggestion
    * 하지만 경로 기반 라우팅 등 간단한 웹 응용 프로그램을 구축할 수 있는 도구를 제공합니다.
```

File: docs/ko/docs/benchmarks.md (hunk 2 above)
Comment (JaeHyuckSa) [inline docs/ko/docs/benchmarks.md:31]:
```suggestion
    * FastAPI는 Starlette에 추가적으로 더 많은 기능을 제공합니다. API를 구축할 때 거의 항상 필요한 데이터 검증 및 직렬화와 같은 기능들이 포함되어 있습니다. 그리고 이를 사용하면 자동 문서화 기능도 제공됩니다(자동 문서화는 응용 프로그램 실행 시 오버헤드를 추가하지 않고 시작 시 생성됩니다).
```

File: docs/ko/docs/benchmarks.md [hunk 5]
```diff
@@ -0,0 +1,34 @@
...
+    * Starlette를 비교할 때는 Sanic, Flask, Django 등의 웹 프레임워크(또는 마이크로 프레임워크)와 비교하세요.
+* **FastAPI**:
+    * Starlette가 Uvicorn을 사용하고 Uvicorn보다 빨라질 수 없는 것과 마찬가지로, **FastAPI**는 Starlette를 사용하므로 더 빠를 수 없습니다.
+    * FastAPI는 Starlette 위에 더 많은 기능을 제공합니다. API를 구축할 때 거의 항상 필요한 데이터 검증 및 직렬화와 같은 기능이 포함되어 있습니다. 그리고 이를 사용하면 자동 문서화 기능도 제공됩니다(자동 문서화는 애플리케이션 실행 시 오버헤드를 추가하지 않고 시작 시 생성됩니다).
+    * FastAPI를 사용하지 않고 직접 Starlette(또는 Sanic, Flask, Responder 등)를 사용했다면 데이터 검증 및 직렬화를 직접 구현해야 합니다. 따라서 최종 애플리케이션은 FastAPI를 사용한 것과 동일한 오버헤드를 가지게 될 것입니다. 많은 경우 데이터 검증 및 직렬화가 애플리케이션에서 작성된 코드 중 가장 많은 부분을 차지합니다.
```
Comment (JaeHyuckSa) [inline docs/ko/docs/benchmarks.md:32]:
```suggestion
    * FastAPI를 사용하지 않고 직접 Starlette(또는 Sanic, Flask, Responder 등)를 사용했다면 데이터 검증 및 직렬화를 직접 구현해야 합니다. 따라서 최종 응용 프로그램은 FastAPI를 사용한 것과 동일한 오버헤드를 가지게 될 것입니다. 많은 경우 데이터 검증 및 직렬화가 응용 프로그램에서 작성된 코드 중 가장 많은 부분을 차지합니다.
```

File: docs/ko/docs/benchmarks.md [hunk 6]
```diff
@@ -0,0 +1,34 @@
...
+    * 따라서 FastAPI를 사용함으로써 개발 시간, 버그, 코드 라인을 줄일 수 있으며, FastAPI를 사용하지 않았을 때와 동일하거나 더 나은 성능을 얻을 수 있습니다(코드에서 모두 구현해야 하기 때문에).
+    * FastAPI를 비교할 때는 Flask-apispec, NestJS, Molten 등 데이터 검증, 직렬화 및 문서화가 통합된 자동 데이터 검증, 직렬화 및 문서화를 제공하는 웹 애플리케이션 프레임워크(또는 도구 세트)와 비교하세요.
```
Comment (JaeHyuckSa) [inline docs/ko/docs/benchmarks.md:34]:
```suggestion
    * FastAPI를 비교할 때는 Flask-apispec, NestJS, Molten 등 데이터 검증, 직렬화 및 문서화가 통합된 자동 데이터 검증, 직렬화 및 문서화를 제공하는 웹 응용 프로그램 프레임워크(또는 도구 집합)와 비교하세요.
```

**Review by JaeHyuckSa — CHANGES_REQUESTED** [review]
Hello! @Limsunoh, thank you for your review. 👍 There are a few revisions needed. (Related issue: https://github.com/fastapi/fastapi/issues/2017)

**Review by JaeHyuckSa — APPROVED** [review]

File: docs/ko/docs/benchmarks.md [hunk 7]
```diff
@@ -0,0 +1,34 @@
...
+* **Uvicorn**:
+    * 서버 자체 외에는 많은 추가 코드가 없기 때문에 최고의 성능을 발휘합니다.
+    * 직접 Uvicorn으로 응용 프로그램을 작성하지는 않을 것입니다. 즉, 사용자의 코드에는 적어도 Starlette(또는 **FastAPI**)에서 제공하는 모든 코드가 포함되어야 합니다. 그렇게 하면 최종 응용 프로그램은 프레임워크를 사용하고 앱 코드와 버그를 최소화하는 것과 동일한 오버헤드를 갖게 됩니다.
+    * Uvicorn을 비교할 때는 Daphne, Hypercorn, uWSGI 등의 응용 프로그램 서버와 비교하세요.
+* **Starlette**:
+    * Uvicorn 다음으로 좋은 성능을 발휘합니다. 사실 Starlette는 Uvicorn을 사용하여 실행됩니다. 따라서 더 많은 코드를 실행해야 하기 때문에 Uvicorn보다 "느려질" 수밖에 없습니다.
+    * 하지만 경로 기반 라우팅 등 간단한 웹 응용 프로그램을 구축할 수 있는 도구를 제공합니다.
+    * Starlette를 비교할 때는 Sanic, Flask, Django 등의 웹 프레임워크(또는 마이크로 프레임워크)와 비교하세요.
+* **FastAPI**:
+    * Starlette가 Uvicorn을 사용하고 Uvicorn보다 빨라질 수 없는 것과 마찬가지로, **FastAPI**는 Starlette를 사용하므로 더 빠를 수 없습니다.
```
Comment (sodaMelon) [inline docs/ko/docs/benchmarks.md:30]:
```suggestion
    * Starlette가 Uvicorn을 사용하므로 Uvicorn보다 빨라질 수 없는 것과 마찬가지로, **FastAPI**는 Starlette를 사용하므로 더 빠를 수 없습니다.
```
동치가 되서, 문장이 명확해지도록 고쳐썼어요.

File: docs/ko/docs/benchmarks.md [hunk 8]
```diff
@@ -0,0 +1,34 @@
...
+    * FastAPI는 Starlette에 추가적으로 더 많은 기능을 제공합니다. API를 구축할 때 거의 항상 필요한 데이터 검증 및 직렬화와 같은 기능들이 포함되어 있습니다. 그리고 이를 사용하면 자동 문서화 기능도 제공됩니다(자동 문서화는 응용 프로그램 실행 시 오버헤드를 추가하지 않고 시작 시 생성됩니다).
```
Comment (sodaMelon) [inline docs/ko/docs/benchmarks.md:31]:
```suggestion
    * FastAPI는 Starlette에 추가적으로 더 많은 기능을 제공합니다. API를 구축할 때 거의 항상 필요한 데이터 검증 및 직렬화와 같은 기능들이 포함되어 있습니다. 그리고 이를 사용하면 문서 자동화 기능도 제공됩니다(문서 자동화는 응용 프로그램 실행 시 오버헤드를 추가하지 않고 시작 시 생성됩니다).
```
automatic documentation = 문서 자동화 (in korea)

**Review by sodaMelon — COMMENTED** [review]
hi.  it's almost perfect
(i left some comments. If you think it's good, reflect it.)

**Comment by sodaMelon** [comment]
@JaeHyuckSa hi.  for application, can we use ``응용 프로그램`` with ``애플리케이션``?
``응용 프로그램`` uses in 7 files 
``애플리케이션`` uses in 12 files.

**Comment by JaeHyuckSa** [comment]
> @JaeHyuckSa hi. for application, can we use `응용 프로그램` with `애플리케이션`? `응용 프로그램` uses in 7 files `애플리케이션` uses in 12 files.

Hello @sodaMelon, we refer to the following link when translating into Korean, instead of basing on usage frequency:

https://github.com/fastapi/fastapi/issues/2017

In the case of "application," it can be translated as "응용 프로그램." Please refer to the link below:

https://www.flowdas.com/pages/python-docs-ko.html#:~:text=application%3A%20%EC%9D%91%EC%9A%A9%20%ED%94%84%EB%A1%9C%EA%B7%B8%EB%9E%A8

For consistency, instances of the word "어플리케이션" across multiple files should be modified accordingly.

**Comment by sodaMelon** [comment]
> > @JaeHyuckSa hi. for application, can we use `응용 프로그램` with `애플리케이션`? `응용 프로그램` uses in 7 files `애플리케이션` uses in 12 files.
> 
> Hello @sodaMelon, we refer to the following link when translating into Korean, instead of basing on usage frequency:
> 
> #2017
> 
> In the case of "application," it can be translated as "응용 프로그램." Please refer to the link below:
> 
> https://www.flowdas.com/pages/python-docs-ko.html#:~:text=application%3A%20%EC%9D%91%EC%9A%A9%20%ED%94%84%EB%A1%9C%EA%B7%B8%EB%9E%A8
> 
> For consistency, instances of the word "어플리케이션" across multiple files should be modified accordingly.

oh thanks. i understand

File: docs/ko/docs/benchmarks.md (hunk 8 above)
Comment (Limsunoh) [reply docs/ko/docs/benchmarks.md]: Thank you. I'm just taking my first steps as a junior developer. I appreciate your help.

**Review by Limsunoh — COMMENTED** [review]
Thank you so much for your help.

**Review by sodaMelon — APPROVED** [review]
LGTM

**Comment by alejsdev** [comment]
Thanks so much for your interest in contributing to FastAPI! @Limsunoh @sodaMelon @JaeHyuckSa :rocket:

**Pull request merged**